base/database/utils.go

```diff
@@ -30,7 +30,7 @@ func Packages(tx *gorm.DB) *gorm.DB {
 		Joins("JOIN package_name pn on p.name_id = pn.id").
 		Joins("JOIN (SELECT id, value FROM strings) descr ON p.description_hash = descr.id").
 		Joins("JOIN (SELECT id, value from strings) sum ON p.summary_hash = sum.id").
-		Joins("JOIN (SELECT id, name, public_date from advisory_metadata) am ON p.advisory_id = am.id")
+		Joins("LEFT JOIN (SELECT id, name, public_date from advisory_metadata) am ON p.advisory_id = am.id")
```

Comment:
@semtexzv why do we have these subselects here? Cannon we simply use "strings desce, strings sum"? Investují tried that and it works. Cca. @MichaelMraka .

Reply:
It should not matter. If you try explain on both variants it should be the same.
But I agree simple `JOIN strings desc ON ...` would be enough and more readable.

Reply:
Ok, so I will make a pr to update it.